main/java/com/example/instagramhomework/Posting.java

```diff
@@ -0,0 +1,44 @@
+package com.example.instagramhomework;
+
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import javax.xml.crypto.Data;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+@WebServlet("/uploadPost")
+public class Posting extends HttpServlet{
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String photo = request.getParameter ("photoUser");
+        String text = request.getParameter("textUser");

```

Comment:
The data shouldn't be an input in the URL parameter it should be sent as JSON in the request's body.